### PR TITLE
Fixes #4595 - Updated nearby query

### LINKED ISSUE #4595: Faster SPARQL query (thanks to wikibase:label)
This query seems to be quite faster than what we have right now.
I works in Amsterdam with a 1km radius (6000 places) in 15 seconds.
```
SELECT
  ?item
  (SAMPLE(?label) AS ?label)
  (SAMPLE(?description) AS ?description)
  (SAMPLE(?class) AS ?class)
  (SAMPLE(?classLabel) AS ?classLabel)
  (SAMPLE(?pic) AS ?pic)
  (SAMPLE(?destroyed) AS ?destroyed)
  (SAMPLE(?endTime) AS ?endTime)
  (SAMPLE(?wikipediaArticle) AS ?wikipediaArticle)
  (SAMPLE(?commonsArticle) AS ?commonsArticle)
  (SAMPLE(?commonsCategory) AS ?commonsCategory)
  (SAMPLE(?monument) AS ?monument)
WHERE {
  # Around given location
  SERVICE wikibase:around {
    ?item wdt:P625 ?location.
    bd:serviceParam wikibase:center "Point(4.89 52.37)"^^geo:wktLiteral. # Longitude latitude
    bd:serviceParam wikibase:radius "1". # Radius in kilometers.
  }

  OPTIONAL {
  ?item p:P31/ps:P31 ?class.
  }

  # Get picture
  OPTIONAL {?item wdt:P18 ?pic}

  # Get existence
  OPTIONAL {?item wdt:P576 ?destroyed}
  OPTIONAL {?item wdt:P582 ?endTime}

  # Get Commons category
  OPTIONAL {?item wdt:P373 ?commonsCategory}

  # Get Wikipedia article
  OPTIONAL {
    ?wikipediaArticle schema:about ?item.
    ?wikipediaArticle schema:isPartOf <https://en.wikipedia.org/>. # TODO internationalization
  }

  # Get Commons article
  OPTIONAL {
    ?commonsArticle schema:about ?item.
    ?commonsArticle schema:isPartOf <https://commons.wikimedia.org/>.
  }

  # Wiki Loves Monuments
  OPTIONAL {?item p:P1435 ?monument}
  OPTIONAL {?item p:P2186 ?monument}
  OPTIONAL {?item p:P1459 ?monument}
  OPTIONAL {?item p:P1460 ?monument}
  OPTIONAL {?item p:P1216 ?monument}
  OPTIONAL {?item p:P709 ?monument}
  OPTIONAL {?item p:P718 ?monument}
  OPTIONAL {?item p:P5694 ?monument}
  
  # Labels
  SERVICE wikibase:label { bd:serviceParam wikibase:language "en,[AUTO_LANGUAGE]". 
                           ?item rdfs:label ?label .
                           ?item schema:description ?description. 
                           ?class rdfs:label ?classLabel . }
}
GROUP BY ?item
```
I suggest implementing it as a quick and easy fix before #4560.

### PR DESCRIPTION
**Pull request by ashishkumar468**
**Description (required)**

Fixes #4595

What changes did you make and why?

Updated the query for nearby to make it execute faster

**Tests performed (required)**

Tested betaDebug on API 29
- Verified that nearby pins show up as expected
-  Verified that monuments show up as expected
-  Made a test upload via nearby and seems to work as expected

### CODE REVIEW

**Comment by ashishkumar468** [comment]
https://user-images.githubusercontent.com/17375274/132803891-c5998751-eebc-4b7a-8da0-80e5b7290efe.mp4

File: app/src/main/resources/queries/monuments_query.rq [hunk 1]
```diff
@@ -1,60 +0,0 @@
-SELECT
```
Comment (ashishkumar468) [inline app/src/main/resources/queries/monuments_query.rq:1]: This file is not being used any more

**Comment by misaochan** [comment]
Tested on real device running Android 11, works well for me! Thanks @ashishkumar468 . :)

Ready to merge pending @nicolas-raoul 's review.

**Review by nicolas-raoul — APPROVED** [review]
Works great! It is much more usable at this speed.

Indentation is slightly off at lines 23 and 24 but that can be dealt with later :-)

**Comment by misaochan** [comment]
Thanks @nicolas-raoul ! We should be able to get the release out on Monday, hopefully with a couple of crash fixes too.

**Pull request merged**